pkg/network/prometheus.go

```diff
@@ -49,6 +49,14 @@ var (
 			Namespace: "neogo",
 		},
 	)
+
+	blockSyncQueueLength = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Help:      "Block state sync queue length",
```

Comment:
What's the difference with `blockQueueLength`?

Reply:
Not much except that block sync queue can only be used during state sync process and it's a separate queue. Fixed (the commit is dropped).